tests/php/test-woocommerce-connect-tracks.php

```diff
@@ -26,8 +26,7 @@ public function test_no_jetpack() {
 		$this->logger->expects( $this->once() )
 			->method( 'log' )
 			->with(
-				$this->stringContains( 'Error' ),
-				$this->anything()
+				$this->stringContains( 'Error. jetpack_tracks_record_event is not defined.' )
```

Comment:
`logger->log()` function only takes 1 parameter. The 2nd is not needed. Fixing this in this PR as well. This fix should resolve this issue https://github.com/Automattic/woocommerce-services/pull/2216#issuecomment-708064854.